src/DynamoCoreWpf/ViewModels/PackageManager/Docs/PublishPackageDocumentation.html

```diff
@@ -217,10 +209,19 @@ <h4>Package Name:</h4>
     </p>
     <p>
         <b>Select Node Libraries:</b><br />
-        Select the node libraries/DLLs from the list of added files.
+    <p>
+        Specify which of the files you added to the package are the node libraries. Node libraries contain the types and methods that Dynamo will import as nodes.
+    </p>
+    <p>
+        It's also valid to mark an assembly which only contains UI NodeCustomization classes as a Node Library.
```

Comment:
Should we say it is `valid` or should we say it is `necessary` to mark such assemblies as Node libraries?

Reply:
Should we also list all other assemblies that should be marked as node libraries or is that stated elsewhere already?

Reply:
This is the only documentation I could find on `node_libraries` within Dynamo - I couldn't find anything in the dev docs or the primer.

I've tried to describe the valid use cases above (ZT OR Nodemodel&NodeViewCustomization OR  Nodemodel OR NodeViewCustomization )

can you suggest a specific improvement?